sopel/plugins/__init__.py

```diff
@@ -80,6 +83,18 @@ def find_sopel_modules_plugins():
             yield handlers.PyModulePlugin(name, 'sopel_modules')
 
 
+def find_entry_point_plugins(group='sopel.plugins'):
+    """List plugins from a setuptools entry point group
+
+    :param str group: setuptools entry point group to look for
+                      (default to ``sopel.plugins``)
```

Comment:
I think we should do our best to stick to this verb tense in these "default value" notes. Yes, I'll probably miss some, and (unlike this instance) not catch them before merging PRs. :laughing:

```suggestion
                      (defaults to ``sopel.plugins``)
```